convert.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python
```

Comment:
I'm not sure `sh` should be the appropriate extension for this file, why not call it `.py`?